taworks/taform/static/taform/css/style.css

```diff
@@ -102,3 +102,96 @@ textarea {
     visibility: hidden;
 }
 
+/*-------------------------
+    Inline help tip
+--------------------------*/
+
+
+.help-tip{
+    position: absolute;
+/*    top: 18px;
+    right: 18px;*/
+    text-align: center;
+    background-color: #A9A9A9;
+    border-radius: 50%;
+    width: 12px;
+    height: 12px;
+    line-height: 12px;
+    cursor: default;
+}
+
+.help-tip:before{
```

Comment:
whats the difference between 

.help-tip

and

.help tip p?

which one gets used when you call class=help-tip?

Reply:
yep crystal